flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1270,11 +1445,6 @@ class BlockDeviceCalculatorTests(TestCase):
     """
     Tests for ``BlockDeviceCalculator``.
     """
-    def setUp(self):
-        super(BlockDeviceCalculatorTests, self).setUp()
-        self.deployer = create_blockdevicedeployer(self)
-        self.persistent_state = InMemoryStatePersister()
```

Comment:
:question: If they aren't being set as ivars in setUp can they stop being used as
ivars and used as method parameters instead?